providers/amazon/tests/unit/amazon/aws/triggers/test_sqs.py

```diff
@@ -37,9 +37,12 @@
 TEST_BOTOCORE_CONFIG = {"region_name": "us-east-1"}
 
 
+@pytest.mark.skipif(not AIRFLOW_V_3_0_PLUS, reason="Requires Airflow 3.0.+")
```

Comment:
Why do we need to skip this test? `SqsSensorTrigger` (I mean using directly the trigger and not through the common messaging) should work in AF2

Reply:
Good point. I am not sure why I put it there - possibly I was over-cautious or mistakenly added it. Let me remove it and see what happeens.